app/src/state/editor/selection.spec.ts

```diff
@@ -0,0 +1,119 @@
+import { editorDefaults, EditorState, Range } from './types';
```

Comment:
The commit is named `🧪️ add tests for editor.go{Left,Right}` but I only see tests for select{Left,Right}. Am I missing something?

Reply:
ah right thats simply a wrong commit message